DangerRulesTest/ViewController.swift

```diff
@@ -10,6 +10,8 @@ import UIKit
 class ViewController: UIViewController {
     
     var testVar:String = "abc"
+    
+    var testVarTwo:Int = 123
 
     override func viewDidLoad() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayur-gauswami/DangerIntegration/blob/e1e2d2fc76ba78d544509e9d920e5b42a9ef7b42/DangerRulesTest/ViewController.swift#L16"></span>Remove overridden functions that don't do anything except call their super<br />
<code>unneeded_override</code> <code>ViewController.swift:16</code></td>
    </tr>
  </tbody>
</table>